middleware/fastcgi/fcgiclient.go

```diff
@@ -380,7 +380,10 @@ func (c *FCGIClient) Request(p map[string]string, req io.Reader) (resp *http.Res
 		if err != nil {
 			return
 		}
-		resp.Status = statusParts[1]
+		if len(statusParts) > 0 {
```

Comment:
nope. > 1 = 2 we are not looking for statusParts[2] - we just need 1

--  
Slawomir Jasinski
e: slav123@gmail.com
    | Quote: "When all else fails read the manual"


On Friday, 12 June 2015 at 11:46 am, Tw wrote:

> In middleware/fastcgi/fcgiclient.go (https://github.com/mholt/caddy/pull/131#discussion_r32285344):
> > @@ -380,7 +380,10 @@ func (c *FCGIClient) Request(p map[string]string, req io.Reader) (resp *http.Res > if err != nil { > return > } > - resp.Status = statusParts[1] > + if len(statusParts) > 0 {  
> I think it should be: if len(statusParts) > 1
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/mholt/caddy/pull/131/files#r32285344).
>  
>  
>  
>  
>